src/rules/no-unlocalized-strings.ts

```diff
@@ -18,7 +18,15 @@ import {
 } from '../helpers'
 import { createRule } from '../create-rule'
 import * as micromatch from 'micromatch'
-import { TypeFlags, UnionType, Type, Expression } from 'typescript'
+import type { UnionType, Type, Expression } from 'typescript'
```

Comment:
If this is conditional, wouldn't you just use the return value from `optionalTypeScript` to get these values instead importing both ways?